app/controllers/sessions_controller.rb

```diff
@@ -2,4 +2,5 @@
 
 class SessionsController < Devise::SessionsController
   before_action :load_game, :load_message_count, except: %i[new destroy]
+  before_action { @game = Game.instance }
```

Comment:
- Does this cause any problems when there is no game created? 
 - Do we have a test to verify that the page still loads when no game is created?
 - This will load the game twice on all the other `sessions_controller`
 - Instead of adding this line can we remove `new` from the `except` block above?

Reply:
> Does this cause any problems when there is no game created?

It does not

> Do we have a test to verify that the page still loads when no game is created?

``test/controllers/session_controller_test 'new user sessions'`` should because no game is defined.

>This will load the game twice on all the other ``sessions_controller``

Fixed

> Instead of adding this line can we remove new from the except block above?

See new solution

Reply:
The 1:load_game1 function on the previous line should take care of this without having to create another method at all, you would just have to change 
```ruby
  before_action :load_game, :load_message_count, except: %i[new destroy]
```
to
```ruby
  before_action :load_game, :load_message_count, except: %i[destroy]
```

Reply:
``load_game`` will check if the game exists, and if not, redirect to the login page which will create an infinite loop.

Reply:
:/ yeah that makes sense, okay in that case can you name this `load_game_for_login` so it is a little more descriptive and also add `def new; end` to the sessions_controller to appease rubocop? May actually need to be

```
def new
  super
end
```

I'm not sure which without trying it.